bundles/framework/divmanazer/component/VisualizationForm.js

```diff
@@ -184,7 +208,8 @@ Oskari.clazz.define(
                     },
                     shape: values.dot.shape,
                     size: values.dot.size
-                }
+                },
+                name: values.name
```

Comment:
This is an undocumented extension to Oskari.style. Including name in the style definition requires further discussion. Adding a new function `getStyleName` would be a better option I think.

Reply:
Removed name from Oskari style by creating styleWithMetadata wrapper which holds name and oskari style. Mentioned wrapper objects are now stored to UserstyleService which holds state of view.